packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -4804,6 +4804,10 @@ export default class Meeting extends StatelessWebexPlugin {
             mediaContent,
           }
         );
+
+        if (mediaContent === 'MAIN') {
```

Comment:
got it

Reply:
@marcin-bazyl since the internal-media-core PR has been merged, will this be updated?

Reply:
@marcin-bazyl ignore that comment, i think u have just been reading my mind 😆

Reply:
yeah, I've just pushed the commits with the update. but after doing more testing I realized that I can't use the undefined values for numTotalSources and numLiveSources and instead I've added an explicit flag to MediaRequestManager options to say if we want trimming or not - it's needed because otherwise we were sending a request for 6 streams in grid layout before we got any VIDEO_SOURCES_COUNT_CHANGED event. So as a result now I don't actually use the MediaContent enum anymore.

Reply:
sorry, made a mistake, I still need to use the MediaContent enum, because we need to call `setNumCurrentSources()` on the right MediaRequestManager (only for main, not for slides), so I've fixed that now.